webpack/JobWizard/autofill.js

```diff
@@ -48,8 +48,11 @@ export const useAutoFill = ({
           })
         );
       }
-      if (search && !hostIds?.length) {
-        setHostsSearchQuery(search);
+      if ((search || search === '') && !hostIds?.length) {
+        // replace an empty string search with a dummy search query to match all hosts
+        // but only if search query was entered (based on presence of :search parameter)
+        const hostSearch = search === '' ? "name != ''" : search;
```

Comment:
Could we somehow not search at all instead of searching using a search query that matches everything?

Reply:
No because we dont want to go back to on default all hosts selected I assume?

Reply:
Right, nevermind then